package.json

```diff
@@ -1834,6 +1834,27 @@
 						"scope": "window",
 						"order": 40
 					},
+					"gitlens.sortRepositoriesBy": {
+						"type": "string",
+						"starredFirst": "lastFetched:desc",
+						"enum": [
+							"name:asc",
+							"name:desc",
+							"lastFetched:asc",
+							"lastFetched:desc",
+							"starredFirst"
```

Comment:
Let's nuke "starredFirst" and just always have starred as the first order of sorting (similar to branches)